docs/datadeling/teknisk-plattform/api/maskinporten.md

```diff
@@ -13,20 +13,18 @@ Dette er standardopsett om man vil opprette en token-tjeneste i API Manager som
 For mer informasjon og forklaring av oppsett [se denne siden](https://www.usit.uio.no/prosjekter/datadeling/arbeidsomrader/integrasjonsarkitektur/dokumentasjon/veiledere/api-manager/jwt-mot-backend.html)
 
 
-**TODO.json**
-
-
 ## Hvordan sette opp API-et
 
 
-1. Sørg for å ha en nøkkel som git tilgng til APIet. Dette kan være virksomhetssertifikat, men vi anbefaler å ha eget nøkkelpar. Se bruksanvisning her
-2. [Anbefalt, emn ike påkrevd] Overfør sertifikatet på en sikker måte til Intark drift. lag zip-fil av nøkkelen og last opp her: TODO
-3. Last ned standardoppsettet over, og opprett tjenesten i API manager. Se [veileder for å registrere en tjeneste i API manager via fil](https://www.usit.uio.no/prosjekter/datadeling/arbeidsomrader/integrasjonsarkitektur/dokumentasjon/veiledere/api-manager/importer-api.html).
-4. Om dette er mot Produksjonsystem i stedet for test, naviger til proxy og endpoint (under backend services) . Klikk på tannhjulet for PROD, fjern haken for backup endpoint. Slett TEST eller huk av for at test skal være backup endpint. Husk å lagre
+1. Sørg for å ha en nøkkel som gir tilgang til APIet. Dette kan være virksomhetssertifikat, men vi anbefaler å ha eget nøkkelpar. Se bruksanvisning her
+2. [Anbefalt, men ikke påkrevd] Overfør sertifikatet på en sikker måte til Intark drift. Lag zip-fil av nøkkelen og kontakt support for å avtale hvordan vi overfører.
```

Comment:
Nemner vi nokon stad å heller sette opp eit eige API for signering, heller enn å overgi virksomhetssertifikatet? Altså slik UiB gjer det.